.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+name: Build 
+on: 
+  push:
+    branches:
+      - staging
+      - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup flutter
+        uses: subosito/flutter-action@v1
+        with:
+          channel: stable
+
+      - name: Build Website
+        working-directory: ./app_widgetbook
+        run: flutter build web -t lib/main.dart
+
+      - name: Upload Widgetbook
+        uses: widgetbook/widgetbook-hosting@0.0.24
+        with:
+          path: 'app_widgetbook/build/web'
+          api_key: 'iL9RQkaGdb3VjLonSu0wb4xlFEKHfErO'
```

Comment:
`widgetbook-hosting` action is obsolete and is replaced by the CLI. See https://pub.dev/packages/widgetbook_cli

Reply:
Created `WIDGETBOOK_CLOUD_API_KEY` secret that contains the newly created key.

Reply:
Thanks. Workflow updated 9aaaa296a650d2a48a106d1a3afbd67491c1d211